lib/src/json_schema/validator.dart

```diff
@@ -37,6 +37,7 @@
 //     THE SOFTWARE.
 
 import 'dart:math';
+import 'package:dart2_constant/convert.dart';
```

Comment:
#nit import styling separation

Reply:
left this here since it's a dart import compatibility package for dart 2.0